packages/block/test/difficulty.spec.ts

```diff
@@ -37,6 +37,9 @@ const hardforkTestData: TestData = {
   arrowGlacier:
     require('../../ethereum-tests/DifficultyTests/dfArrowGlacier/difficultyArrowGlacier.json')
       .difficultyArrowGlacier.ArrowGlacier,
+  grayGlacier:
+    require('../../ethereum-tests/DifficultyTests/dfGrayGlacier/difficultyGrayGlacier.json')
+      .difficultyGrayGlacier.GrayGlacier,
```

Comment:
So to your question with the tests: just copy the whole file over and move to the local Block test data folder and reference through the local path. You can maybe add a TODO here that this should be replaced with the `ethereum-tests` folder again at some point.